File: Plugins/engines/org.eclipse.escriptmonkey.scripting.engine.python.jython.samples/samples/scripts/IO__Web_Sample.py
#
# Thread:UI
# Menu: Examples >IO > Web Sample
# Kudos: Paul Colton, Aptana, Inc.
# Description: You can use this script to display into a dialog a http request result.
# License: EPL 1.0
#

import urllib

import org.eclipse.jface.dialogs

workbench = loadModule("WorkbenchModule")
req = urllib.urlopen("http://xml.weather.yahoo.com/forecastrss?p=94103")
text = req.read()

org.eclipse.jface.dialogs.MessageDialog.openInformation(
	WINDOW.getShell(), 	
	"San Francisco Weather Feed", 
	text
)

File: Plugins/engines/org.eclipse.escriptmonkey.scripting.engine.javascript.rhino.samples/samples/scripts/IO__Web_Sample.js
/*
 * Menu: Examples > IO > JS > Web Sample
 * Kudos: Paul Colton, Aptana, Inc.
 * License: EPL 1.0
 * Description : You can use this script to display into a dialog a http request result.
 * DOM: http://download.eclipse.org/technology/dash/update/org.eclipse.eclipsemonkey.lang.javascript
 */
  
function main() 
{
	var req = new WebRequest();
	
	req.open("GET", "http://xml.weather.yahoo.com/forecastrss?p=94103");
	
	var text = req.send();
	
	Packages.org.eclipse.jface.dialogs.MessageDialog.openInformation( 	
		window.getShell(), 	
		"San Francisco Weather Feed", 
		text	
	);
	
}
